modules/dxp/apps/search-experiences/search-experiences-web/src/main/resources/META-INF/resources/sxp_blueprint_admin/js/edit_sxp_blueprint/EditSXPBlueprintForm.js

```diff
@@ -49,7 +49,7 @@ import {
 	validateNumberRange,
 	validateRequired,
 } from '../utils/validation';
-import AddSXPElementSidebar from './AddSXPElementSidebar';
+import AddSXPElementSidebar from './add_sxp_element_sidebar';
```

Comment:
This import won't resolve in portal. It'll resolve in storybook, but in portal we use our own module resolver so we need the `/index` added to this.

Reply:
Added, thanks for letting me know!